### PR TITLE
Add more concepts to curriculum arc

### PR DESCRIPTION
**Pull request by jonleung**

### CODE REVIEW

File: workshops/ARC.md [hunk 1]
```diff
@@ -44,3 +44,60 @@ for hacking.
 
 [portfolio]: portfolio/README.md
 [twilio]: twilio/README.md
+
+## Concept Arc
+
+- pair programming
```
Comment (paked) [inline workshops/ARC.md]: These all need to be capitalised correctly.

Also:

- `html`->`HTML`
- `dom`->`DOM`
- etc.

Comment (jonleung) [inline workshops/ARC.md]: Fixed!

**Comment by jonleung** [comment]
@paked, any updates on this?

**Comment by jonleung** [comment]
I decided to close this pull request in favor of #682 because the section where I slow the the concepts into workshops, it replaces this section.

**Pull request closed without merging**